appcelerator-archive/amp
# Add API verb service.ServiceScale

closes #1316

Add verb ServiceScale to AMP service API

Use UI to test;
- log in UI
- go to Stack item
- deploy a stack pinger
- see the stack info, click on service
- see the service of the stack
- select it
- clic on Scale button
- enter a number
- see the service updated
